### ISSUE TITLE
Als begraafplaatsbeheerder wil is een unieke nummering van de binnekomende verzoeken, 

### ISSUE THREAD

**Issue by PatrickCastenmiller** [VNG-Realisatie/Begrafenisplanner#146]
zodat ik hier ondubbelzinnig naar kan verwijzen met de begrafenisondernemer en de eigen organisatie 

Het lijkt erop dat er nu dubbele nummers worden uitgegegeven.

- [ ] Zorg dat de beschrijving duidelijk voor iedereen in het team is
- [ ] Koppel de juiste labels
- [ ] Koppel eventueel aan een epic (door # en het nummer toe te voegen)
- [ ] Voeg alle taken die nodig zijn voor de user story toe.

**Comment by rubenvdlinde** [comment]
Know issue, heeft ermee te maken dat we issues overzetten van westfriesland naar de betreffende gemeente, met als gevold dat die referentie dus op west friesland niet meer bestaat.

Gevolg, referentie nummer word op westfriesland opnieuw aangemaakt en dan doorgezet naar de betreffende gemeente.

**Comment by PatrickCastenmiller** [comment]
Ja, interessante complexiteit. Zeker als we landelijk een loket zouden inrichten :-)

Uitgangspunt:
Verzoek is van de klant
De gemeente neemt dit in behandeling
